partiql-parser/src/main/kotlin/org/partiql/parser/internal/PartiQLParserDefault.kt

```diff
@@ -863,7 +863,11 @@ internal class PartiQLParserDefault : PartiQLParser {
         }
 
         override fun visitQueryExpression(ctx: GeneratedParser.QueryExpressionContext) = translate(ctx) {
-            val body = visitAs<QueryBody>(ctx.queryExpressionBody())
+            val body = when (val node = visit(ctx.queryExpressionBody())) {
+                is QueryBody -> node
+                is ExprQuerySet -> node.body
+                else -> throw error(ctx, "Unexpected QueryExpression")
+            }
```

Comment:
self-review: previously, there was a parsing error when there was a `LIMIT`, `OFFSET`, and/or `ORDER BY` on a set operator. New conformance tests added in https://github.com/partiql/partiql-tests/pull/142 now pass.